castle/cms/static/components/analytics-modal.js

```diff
@@ -472,27 +491,26 @@ define([
     },
     renderSocialTab: function(){
       if(this.state.social === null){
-        return D.p({ className: 'discreet' }, 'No social data gathered');
+        return D.p({ className: 'discreet' }, 'No social data found (see also social-counts script)');
       }
       var extra = '';
       var data = this.state.social;
       var cdata= [
         ['Platform', 'Shares'],
-        ['Facebook', data.facebook],
-        ['Linked In', data.linkedin],
-        ['Pinterist', data.pinterist]
+        ['Pinterest', data.pinterest],
+        ['Twitter (Matomo)', data.twitter_matomo],
```

Comment:
I wonder if we should conditionally show these?

Not sure it's great that everyone will see "Matomo" in the UI. Doesn't seem like very many use this feature either so maybe it doesn't matter.